src/file-2.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:25919/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:25919/project/issues?id=sample&branch=feature1&issues=AWFL2-YHAL-TMG0S82x5&open=AWFL2-YHAL-TMG0S82x5)

[ISSUE_KEY]: AWFL2-YHAL-TMG0S82x5

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED